app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,87 @@
  *= require_tree .
```

Comment:
issue: If you are using scss stylesheets in your app, it's best, though obviously not required, to make all of them scss stylesheets. To do that here:

1. Rename this file to `application.scss`
1. Remove `*= require_tree .` and `*= require_self`
1. Import bootstrap styles first: `@import "bootstrap";`
1. Explicitly import your other scss files. Example: https://github.com/pulibrary/orangelight/blob/main/app/assets/stylesheets/application.scss#L36-L41
1. Move the css styles in lines 17-99 to another file or files (grouped on their use) and import them.